site/content/docs/latest/tutorials/kubeapps-on-tce/README.md

```diff
@@ -0,0 +1,7 @@
+# Deploy and Configure Kubeapps on VMware Tanzu™ Community Edition
+
+[VMware Tanzu™ Community Edition (TCE)](https://tanzucommunityedition.io) is a full-featured, easy-to-manage Kubernetes platform for learners and users, especially those working in small-scale or preproduction environments.
+
+[Kubeapps](https://kubeapps.com/) provides a web-based dashboard to deploy, manage, and upgrade applications on a Kubernetes cluster. It is a one-time install that gives you a number of important benefits, including the ability to browse and deploy Helm charts from public or private chart repositories and upgrade, manage and delete deployments from your Web browser.
```

Comment:
>It is a one-time install that gives you a number of important benefits, including the ability to browse and deploy Helm charts from public or private chart repositories and upgrade, manage and delete deployments from your Web browser. 

Rephrase this sentence, too much marketing here. Also add Carvel and Flux

Reply:
It is an exact copy paste of what the TKG tutorial says [here](https://github.com/vmware-tanzu/kubeapps/blob/main/site/content/docs/latest/tutorials/kubeapps-on-tkg/README.md).